Guess game/Program.cs

```diff
@@ -1,89 +1,126 @@
         }
-        
+
         static int StartGame()
         {
             int progress = 0;
-            int guessLimit = 3;
 
-                foreach (KeyValuePair<string, string> riddle in Riddles)
+            foreach (KeyValuePair<string, string> riddle in Riddles)
+            {
+                int guessCount = 0;
+                Console.WriteLine("------------------------------------------------------------------------------");
+                Console.WriteLine(riddle.Key);
+
+                while (guessCount < GuessLimit)
                 {
-                    int guessCount = 0;
-                    Console.WriteLine(riddle.Key);
+                    Console.WriteLine();
+                    Console.Write("Įveskite savo spėjimą: ");
+                    string guess = Console.ReadLine();
+
+                    if (guess == riddle.Value)
+                    {
+                        GiveAnswer(correct: true);
+                        progress += GuessLimit - guessCount;
+                        break;
+                    }
 
-                    while (guessCount < guessLimit)
+                    if (guessCount == GuessLimit - 1 & guess != riddle.Value)
                     {
-                        Console.Write("Enter your guess ");
-                       string guess = Console.ReadLine();
-
-                        if (guess == riddle.Value)
-                        {
-                            GiveAnswer(correct: true);
-                            progress += guessLimit - guessCount;
-                            break;
-                        }
-
-                        if (guessCount == guessLimit - 1 & guess != riddle.Value)
-                        {
-                            GiveAnswer(correct: false);
-                            break;
-                        }
-
-                        guessCount++;
+                        GiveAnswer(correct: false, lastAttempt: true);
+                        break;
                     }
+
+                    GiveAnswer(correct: false);
+                    guessCount++;
                 }
+            }
 
             return progress;
         }
 
-        private static void GiveAnswer(bool correct)
+        private static void GiveAnswer(bool correct, bool lastAttempt = false)
         {
-            Console.WriteLine(correct ? "Nice you get it " : "You did not get it ");
+            if (correct)
+            {
+                GiveAnswerColor(rightAnswerColor: true);
+            }
+            else if (!lastAttempt)
+            {
+                GiveAnswerColor(rightAnswerColor: false);
+            }
+            else
+            {
+                GiveAnswerColor(rightAnswerColor: false, wrongAnswerColor: true);
+            }
+        }
 
-            Console.Write("Press Enter to continue ");
-            Console.ReadLine();
+        private static void GiveAnswerColor(bool rightAnswerColor, bool wrongAnswerColor = false)
```

Comment:
you don;t need to implement GiveAnswer IF's two times, so this extra method `GiveAnswerColor` is not needed. You can do just this:
```
private static void GiveAnswer(bool correct, bool lastAttempt = false)
{
    if (correct)
    {
        WriteColoredMessage("Teisingai!", ConsoleColor.DarkGreen);
    }
    else if (!lastAttempt)
    {
        WriteColoredMessage("Neteisingai. Bandykite dar kart!", ConsoleColor.DarkYellow);
    }
    else
    {
        WriteColoredMessage("Neteisingai. Visi spėjimai išnaudoti!", ConsoleColor.DarkRed);
    }
}

private static void WriteColoredMessage(string message, ConsoleColor color)
{
    Console.ForegroundColor = color;
    Console.WriteLine(message);
    Console.ResetColor();
}
```